apps/web/src/index.css

```
@@ -15,7 +15,9 @@
 @layer components {
   /* ─── Editor content ─── */
   .editor-content {
-    @apply outline-none min-h-[480px] text-[16px] leading-[1.75] text-notion-text;
+    @apply outline-none min-h-[480px] text-[16px] leading-[1.75] text-notion-text w-full;
+    word-break: break-word;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)